frontend/src/entities/enemies/Cow.js

```diff
@@ -0,0 +1,11 @@
+import Enemy from "./Enemy";
+
+export default class Cow extends Enemy {
+    constructor (x, y, z, scale) {
+        super(x, y, z, scale ?? 0.01);
+        this.name = "cow";
+        this.maxHp = 200;
+        this.hp = this.maxHp;
+        this.speed = 0.5;
```

Comment:
Is this actually used? Don't see it for the other enemies.

Reply:
Nope, it's moved to a static in my next pr, so I'll remove it here.